ssz/src/main/java/tech/pegasys/artemis/ssz/backing/tree/TreeNode.java

```diff
@@ -32,7 +31,7 @@
 public interface TreeNode {
 
   static TreeNode createLeafNode(Bytes32 val) {
-    return new LeafNodeImpl(val);
+    return new TreeNodeImpl.LeafNodeImpl(val);
```

Comment:
nit: May as well keep the import for `LeafNodeImpl`.

Reply:
Done.